spec/page_objects/pages/jobseeker/job_applications/self_disclosure/date_field.rb

```diff
@@ -0,0 +1,21 @@
+# frozen_string_literal: true
+
+module PageObjects
+  module Pages
+    module Jobseeker
+      module JobApplications
+        module SelfDisclosure
+          class DateField < SitePrism::Section
```

Comment:
as this isn't a page, does it belong in a different namespace to PageObjects::Pages?